src/map/buildDetailView.js

```diff
@@ -201,7 +201,7 @@ export default (
   const layerObjDatum =
     activeData &&
     activeData.length &&
-    activeData.find(d => (d.properties || d)[join[1]] === featureProperties[join[0]]);
+    activeData.find(d => (d.properties || d)[join && join[1]] === featureProperties[join && join[0]]);
```

Comment:
Yeah that's a good catch, is there anywhere else in the codebase we're not already checking for join?

Reply:
@cKellyDesign not sure whether similar cases exist. I spotted this issue when fixing an issue on somnutrition sites